### PR TITLE
Fix some mismatched HTML tags

### PR DESCRIPTION
**Pull request by robertknight**
Fix some HTML tag nesting errors reported when testing a Jinja template formatter [1] on h's Jinja templates.

[1] https://github.com/davidodenwald/prettier-plugin-jinja-template

### CODE REVIEW

File: h/templates/activity/search.html.jinja2 [hunk 1]
```diff
@@ -207,7 +207,6 @@
       </li>
     {% endfor %}
     </ul>
-  </form>
```
Comment (robertknight) [inline h/templates/activity/search.html.jinja2:210]: There is no corresponding `<form>`.

File: h/templates/admin/badge.html.jinja2 [hunk 2]
```diff
@@ -19,7 +19,6 @@
     you can backslash-escape it in your pattern: <code>\_</code>.
   </p>
 
-  </ul>
```
Comment (robertknight) [inline h/templates/admin/badge.html.jinja2:22]: There is no containing `<ul>`

File: h/templates/activity/search.html.jinja2 [hunk 3]
```diff
@@ -222,7 +221,7 @@
       <div class="search-result-sidebar__organization">
         <span class="search-result-sidebar__org-logo">
           {% if organization.logo %}
-            <img src="{{ organization.logo }}" alt="Organization logo" class='search-result-sidebar__logo'></img>
+            <img src="{{ organization.logo }}" alt="Organization logo" class='search-result-sidebar__logo'>
```
Comment (lyzadanger) [inline h/templates/activity/search.html.jinja2:224]: Huh, I realize I'm still a robot when it comes to `img` tags and still close them (`/>`). But hey, you know, HTML5 has been around a while now and I should really get with the proverbial program :).

**Review by lyzadanger — APPROVED** [review]
👍🏻

**Pull request merged**